packaging/func-e.wxs

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- This builds a minimal package of func-e. The only required variable is Version
+     Ex. `wixl -D Version=0.6.1`
+
+     While all options used are documented, the initial version of this file was helped by:
+     * cloudflared - https://github.com/cloudflare/cloudflared/commit/38af26e232d6a5fd981d40981b702a1d06a3b8ee
+     * podmad - https://github.com/containers/podman/commit/b6b0b6e8bd0f8c75fe9411f722a43f9305c6137a
```

Comment:
I would also cite apple's swift, except it uses windows-only (ex non GNOME) features, which actually resulted in me losing time to figure that out. Luckily I commented below which features would be nice if GNOME msitools ends up supporting them https://github.com/apple/swift-installer-scripts/blob/bbcd6a3addfe9c3b62caebf1683cd39c27e7ddb9/platforms/Windows/Toolchain/windows-toolchain.wxs